Sources/CrashReportConverter.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  CrashReportConverter.swift
+//  SentrySwift
+//
+//  Created by Daniel Griesser on 05/11/16.
+//
+//
+
+import Foundation
+
+public class CrashReportConverter {
+    
+    internal func convertReportToEvent(_ report: CrashDictionary) -> Event? {
+        // Extract crash timestamp
+        #if swift(>=3.0)
+            let timestamp: NSDate = {
```

Comment:
There is a date helper already in the project if you want to use that - https://github.com/getsentry/sentry-swift/blob/master/Sources/NSDate%2BExtras.swift